www/js/controllers/epaper-menuctrl.js

```diff
@@ -58,6 +58,203 @@ angular.module('epaper.controllers')
 				//$state.$apply();
             });
 			
+		};
+		
+		// Form data for the login modal
+		$scope.loginData = {};
+		
+		$scope.isAuthenticated = true;
+		$scope.currentUser = localStorage.getItem('shApiUsername');
+		
+		if ($scope.currentUser == undefined) {
+			
+			$scope.currentUser = '访客';
+			$scope.isAuthenticated = false;
+			
+			if (ShApiConstants.useProxy) {
+				$cookies.remove(localStorage.getItem('shApiSessionKey'), {path:'/'});
+			}
+			
+		} else {
+			
+			// verify if session cookies is still available
+			// - if gone need to refresh session data based on localStorage values
+			// - possible scenario: user closed the app intentionally or restart device
+			if (ShApiConstants.useProxy) {
```

Comment:
Can you explain more on the useProxy? It is written in the Constants.js that useProxy is to handle CORS. why we use it to clear the cookies storage?